wolfssl/wolfcrypt/settings.h

```diff
@@ -4538,6 +4538,10 @@ extern void uITRON4_free(void *p) ;
 #error "If TLS is enabled please make sure either client or server is enabled."
 #endif
 
+#ifdef WOLFSSL_DEBUG
```

Comment:
I agree, please remove this logic.

Reply:
Done. I assume `#warning` also undesirable at compile time.

Is there a place to enter `WOLFSSL_DEBUG` as a known spelling error to be caught during Jenkins or GitHub Actions?